### ISSUE TITLE
Can't uninstall HB Service

### ISSUE THREAD

**Issue by philipyoungg** [oznu/homebridge-config-ui-x#1910]
### Describe The Bug

I'm migrating my homebridge to docker. I wanted to uninstall `hb-service` but got issue.

```The 'hb-service uninstall' command is not available when Homebridge is running as a system package```

### Homebridge UI Version

4.53.0

### Homebridge Version

1.7.0

### Node.js Version

18.13.0

### Operating System

Ubuntu / Debian (or a variant)

### Environment Info

- [ ] Using Docker?
- [ ] Using Hyper-V?
- [ ] Using hb-service?

### Raspberry Pi Model

None

**Comment by bwp91** [comment]
Hi @philipyoungg does this comment help at all? I know it's not a direct solution, but maybe it can help you understand how you have homebridge running so that it might be clearer how to properly uninstall/remove hb-service?

https://github.com/homebridge/homebridge-config-ui-x/issues/1573#issuecomment-1758890721

**Comment by philipyoungg** [comment]
Sorry for the late reply. I didn't get notification on Github.

I honestly don't remember how I had it installed in Ubuntu.

I tried following https://github.com/homebridge/homebridge-config-ui-x/wiki/How-To-Swap-From-Standalone-Mode-to-Service-Mode but 

- `Remove the homebridge-config-ui-x service:`  
  - doesn't work because I don't think it was installed as `standalone` back when I tried doing it.
- `Remove the homebridge service:` 
  - this part works.
- `sudo hb-service install --user homebridge --port 8080 -U /var/lib/homebridge` 
  - this part doesn't work. Still the same error as `The 'hb-service install' command is not available when Homebridge is running as a system package.`
